### DISCUSSION TITLE
Accounting for polymorphism on a population level with clear group separation

### DISCUSSION THREAD

**Discussion by JanSchlender** [Open-Systems-Pharmacology/Forum#1511] [Questions & Problems] [answered]
Hi all,

I was wondering about a smart incorporation of polymorphism on a population level. I would like to create equally sized groups for each genetic variant. While I see multiple ways of creating such a population, there are limitations for each approach. 

1. When creating separate populations with different reference concentrations (as described here https://github.com/Open-Systems-Pharmacology/Forum/issues/128), I would like to add variability around this reference concentration. The tails of the each population will overlap with other genetic conditions and, thus, will encounter subjects in one condition which should belong to another condition. The ontogeny factor will even widen this distribution.
2. Creating a uniform distribution for a range, will not help to account for a clustering around the mean reference expression of a polymorphic condition. 

Any good suggestion?

**Comment by PavelBal** [comment]
How about 1., then export to csv, and post-process in R? Assign each individual to one of the populations based on the sampled reference concentration value.

**Comment by JanSchlender** [comment]
Reference concentration * ontogeny factor and then assign to a group. But how to ensure equally sized groups? If you want a (log)normal distribution around each mean, you probably need to create multiple large populations for each polymorphic condition and sample from these populations within each reference concentration range

**Comment by PavelBal** [comment, marked as answer]
Hmm true.

Or create population in PK-Sim WITHOUT variability of reference concentration and/or ontogeny factor, sample "reference concentration" values in R and add to the population?

**Reply by JanSchlender** [reply]
Feature wish :)

Sounds reasonable and does not screw any covariance since the reference concentration of one enzyme is not linked to any other enzyme / parameter (as far as I know).
